datalad/distribution/create_sibling_github.py

```diff
@@ -180,98 +183,22 @@ def __call__(
                 DeprecationWarning)
             dry_run = dryrun
 
-        # this is an absolute leaf package, import locally to avoid
-        # unnecessary dependencies
-        from datalad.support.github_ import _make_github_repos_
-
-        if reponame != normalize_reponame(reponame):
-            raise ValueError('Invalid name for a GitHub project: {}'.format(
-                reponame))
-
-        # what to operate on
-        ds = require_dataset(
-            dataset, check_installed=True, purpose='create GitHub sibling(s)')
-
-        res_kwargs = dict(
-            action='create_sibling_github [dry-run]' if dry_run else
-            'create_sibling_github',
-            logger=lgr,
-            refds=ds.path,
+        # TODO robustify the deprecation mappings
+        auth = auth or github_login
```

Comment:
why `login` should be here? to tell apart one of the credentials from another? (IIRC we had something like that in current implementation for github to support choosing between possibly multiple tokens available)

Reply:
This is purely for compatibility with the present API. We can rip it out.